### ISSUE TITLE
Add Ingestion module to docker

### ISSUE THREAD

**Issue by immangat** [CodeForBc/airbnb-regulation#45]
Currently only the db is in the docker compose. 

The ingestion module should also be inside the compose.

### Acceptance Criteria:
Running only the docker compose should allow the ingestion module to work with the db and policy.

**Comment by VimanyuAgg** [comment]
Is this done with https://github.com/CodeForBc/airbnb-regulation/pull/44 ?

**Comment by immangat** [comment]
Yes, after 44. the ingestion module is in docker and is connected to the db in docker.
